requirements.txt

```diff
@@ -1,27 +1,113 @@
-# dependencies
```

Comment:
@proy9714 hey sorry to nag no need for this conditional file. Simply add the new packages without their version. After that happy to merge, thanks! 👯